src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -485,35 +549,335 @@ .maxstack  1
+                    Diagnostic(ErrorCode.ERR_ConcreteMissingBody, "init").WithArguments("I.Q3.init").WithLocation(5, 39));
+            }
+
+            var actualMembers = comp.GetMember<NamedTypeSymbol>("I").GetMembers().ToTestDisplayStrings();
+            string[] expectedMembers;
+            if (languageVersion == LanguageVersion.CSharp13)
+            {
+                expectedMembers = new[]
+                    {
+                        "System.Object I.Q1 { get; set; }",
+                        "System.Object I.Q1.get",
+                        "void I.Q1.set",
+                        "System.Object I.Q2 { get; set; }",
+                        "System.Object I.Q2.get",
+                        "void I.Q2.set",
+                        "System.Object I.Q3 { get; init; }",
+                        "System.Object I.Q3.get",
+                        "void modreq(System.Runtime.CompilerServices.IsExternalInit) I.Q3.init",
+                    };
+            }
+            else
+            {
+                expectedMembers = new[]
+                    {
+                        "System.Object I.<Q1>k__BackingField",
+                        "System.Object I.Q1 { get; set; }",
+                        "System.Object I.Q1.get",
+                        "void I.Q1.set",
+                        "System.Object I.<Q2>k__BackingField",
+                        "System.Object I.Q2 { get; set; }",
+                        "System.Object I.Q2.get",
+                        "void I.Q2.set",
+                        "System.Object I.<Q3>k__BackingField",
+                        "System.Object I.Q3 { get; init; }",
+                        "System.Object I.Q3.get",
+                        "void modreq(System.Runtime.CompilerServices.IsExternalInit) I.Q3.init",
+                    };
+            }
+            AssertEx.Equal(expectedMembers, actualMembers);
+        }
+
+        [Theory]
+        [CombinatorialData]
+        public void ImplicitAccessorBody_04(
+            [CombinatorialValues("class", "struct", "ref struct", "record", "record struct")] string typeKind,
+            [CombinatorialValues(LanguageVersion.CSharp13, LanguageVersion.Preview)] LanguageVersion languageVersion)
+        {
+            string source = $$"""
+                {{typeKind}} A
                 {
-                  // Code size        8 (0x8)
-                  .maxstack  2
-                  IL_0000:  ldarg.0
-                  IL_0001:  ldarg.1
-                  IL_0002:  stfld      "object I.<Q2>k__BackingField"
-                  IL_0007:  ret
+                    public static int P1 { get; set { } }
+                    public static int P2 { get { return -2; } set; }
+                    public int P3 { get; set { } }
+                    public int P4 { get { return -4; } set; }
+                    public int P5 { get; init { } }
+                    public int P6 { get { return -6; } init; }
```

Comment:
for the future, I would def consider a "half-auto" prop, where the manually implemented accessor never reads `field`, to possibly be a warning scenario.

For example, if you have `get => notField; set;` or `get; set => notField = value`, it practically means you want to ignore the value that was set. There are better ways to do that than introducing a backing field. It very likely means you are making a mistake.

Reply:
There is an [open question](https://github.com/dotnet/csharplang/blob/main/proposals/field-keyword.md#scenarios-similar-to--set-) in the proposal.

Reply:
To obtain these warnings, I added <https://github.com/dotnet/csharplang/blob/main/proposals/field-keyword.md#field-usage-warnings> since the proposal hadn't spoken to this aspect at all yet.